src/components/FeaturedProducts/FeaturedProducts.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './style.scss';
+import { Button } from '@material-ui/core';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import image from './pictureproducts.png';
+
+function FeaturedProducts() {
+  return (
+    <div className="productsContainer">
+      <img src={image} alt="child-stats" />
+      <div className="productsHeader">
+        <span className="redStripe" />
```

Comment:
classes convention if you use scss - as we were talking before. Plus now its redStripe, what if it will be blue one later? use stripe__sth--red and if its unique, change it's name so it will not contain color name. check other places too